docs/tools/aiven-console/howto/create-manage-teams.md

```diff
@@ -72,10 +72,13 @@ clicking the more options menu for the project.
     -   the permission level that is assigned for each project
 
     :::note
-    Users on the Account Owners team automatically become super admin with full access to
-    manage the organization. You don't need to create a group for these users or manage
-    this team after the migration. Instead, you can
-    [make users super admin](/docs/platform/howto/make-super-admin).
+    Users on the Account Owners team are automatically made super admin
+    with full access to manage the organization. You don't need to create a
+    group for these users.
+
+    If you create a group named Account Owners, users you add to that group won't
+    automatically become super admin. Instead, you can
```

Comment:
Would plural make more sense here: "super admins"? And in the line below

Reply:
We use the uncountable noun for admin and super admin.